.github/CODEOWNERS

```diff
@@ -4,9 +4,9 @@
 # For syntax help see:
 # https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
 
-# The @googleapis/api-pubsublite is the default owner for changes in this repo
-*                       @googleapis/yoshi-java @googleapis/api-pubsublite
-**/*.java               @googleapis/api-pubsublite
+# The @googleapis/api-pubsub is the default owner for changes in this repo
+*                       @googleapis/yoshi-java @googleapis/api-pubsub
+**/*.java               @googleapis/api-pubsub
 
```

Comment:
Add the `CODEOWNERS` file to exclude list in synth.py to prevent autosynth from suggesting api-pubsub as the codeowner